### ISSUE TITLE
Add Makefile to automatize the build

### ISSUE THREAD

**Issue by radare** [Yara-Rules/r2yara#3]
Clone yara, copy files, etc

**Comment by plutec** [comment]
What do you mean with "Makefile"? A real Makefile yo use with make or a kind of "makefile.sh"?

**Comment by radare** [comment]
A real makefile or an r2pm package

**Comment by plutec** [comment]
Do you have any kind of tutorial to create a r2pm package?

**Comment by radare** [comment]
nope, its just a shellscript with 2 functions: to install and to uninstall

check the readme and any random file from the db/ directory:

https://github.com/radare/radare2-pm <https://github.com/radare/radare2-pm>
